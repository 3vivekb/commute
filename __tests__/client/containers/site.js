/* global describe, expect, it */

import { mount } from 'enzyme'
import { mountToJson } from 'enzyme-to-json'
import React from 'react'
import { Provider } from 'react-redux'

import { makeMockStore, mockStoreData } from '../../test-utils/mock-store.js'

import Site from '../../../client/containers/site'

const mockStore = makeMockStore(mockStoreData)

describe('Container > Site', () => {
  it('renders correctly', () => {
    // mount component
    const tree = mount(
      <Provider store={mockStore}>
        <Site />
      </Provider>
    )
    expect(mountToJson(tree.find(Site))).toMatchSnapshot()
  })
})
