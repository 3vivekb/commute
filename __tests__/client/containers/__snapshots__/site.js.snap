exports[`Container > Site renders correctly 1`] = `
<Connect(Site)>
  <Site
    dispatch={[Function]}>
    <h1>
      Site
    </h1>
  </Site>
</Connect(Site)>
`;
